extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -414,6 +420,14 @@ export class AmpAnalytics extends AMP.BaseElement {
    * @private
    */
   handleRequestForEvent_(request, trigger, event) {
+    // TODO(avimehta, #6543): Remove this code or mark as "error" for the
+    // once destroyed embed release is implemented. See `detachedCallback`.
+    if (!this.element.ownerDocument.defaultView) {
+      const TAG = this.getName_();
+      dev().warn(TAG, 'request against destroyed embed: ', trigger['on']);
```

Comment:
It'd, but not as noise as the error we are currently sending :) warns actually don't go anywhere unless explicitly enabled.